prql-php/README.md

```diff
@@ -61,3 +61,9 @@ There is a `build.sh` script that:
 sh build.sh
 ./vendor/bin/phpunit tests
 ```
+
+### Code style
+
+```
+./vendor/bin/phpcs --standard=PSR12 src tests
```

Comment:
@eitsupi is this a time we could use MegaLinter!

It might a perfect time for it, because the pre-commit hook requires installing external dependencies, which we wouldn't want for something non-core like PHP: https://github.com/digitalpulp/pre-commit-php#php-cs

Reply:
Yeah, current linter is failing on some cases where it shouldn't: https://github.com/PRQL/prql/pull/2039/files/0113c48639839b0ae2121f1267c2b88b7967cce9..6a33b491a882d24e955a0b61b992aa7dfac08b73

Reply:
They seem to be due to phpstan.
Try disabling these.......